src/unity/schemas/Organization.yml

```diff
@@ -1,23 +1,37 @@
 type: object
 properties:
-  idpeId:
-    type: string
-    description: idpe ID of the org
   id:
-    type: number
-    description: quartz ID of the org
+    type: string
+    description: ID of the org
```

Comment:
```suggestion
    description: IDPE ID of the org
```

#nit

Reply:
Updated in https://github.com/influxdata/openapi/pull/304/commits/0346df21a2f03e7163b1469b1c957660c3903cfc